CHANGELOG.rst

```diff
@@ -10,6 +10,7 @@
 * Dropped deprecated APIs of old makegateway names.
 * Removed ``py`` testing dependency.
 * Explicitly pass ``encoding`` when opening files in the gateway to get rid of warnings when using ``PYTHONWARNDEFAULTENCODING=1`` (#195).
+* Fixed error when loading source code files containing non-ascii characters.
```

Comment:
#201

Reply:
I think non-ascii source code already worked fine; the problem was that we insert the grandparent-directory into `sys.path` and if _that path_ contained non-ascii characters then we'd crash.

Reply:
Ahh thanks!